src/main/resources/static/_shared.yml

```diff
@@ -4319,9 +4319,8 @@ components:
         assessmentId:
           type: string
           format: uuid
-        createdByUserId:
```

Comment:
sanity check - the ui isn't going to explode if this is missing?

Reply:
It might, but I'm expecting we'll fix it fairly quickly

Reply:
ok I guess not an issue if only appearing on appeals pages that i guess aren't shown in prod anyway